src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.search-field {
+  position: relative;
+  display: block;
+}
+
+.form-input {
+  font-family: inherit;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  font-size: 16px;
+  background: url(./images/Search.svg) 26px 25px no-repeat;
+}
+
+.form-input--small {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+  background: url(./images/Search.svg) 11px 11px/13px 15px no-repeat;
+}
+
+.form-input::placeholder {
+  color: #3d4e61;
+}
+
+.form-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-input:focus {
+  border-radius: 3px;
+  outline: none;
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
it is why your icon disappears
you override `background` style for tag

Reply:
Got it, thank you!